test/models/organization_test.rb

```diff
@@ -126,4 +126,17 @@ def organization
   test 'related developments custom method responds correctly' do
     assert_respond_to(org, :developments)
   end
+
+  private
+
+  def webster
+    """
```

Comment:
Combine "" and "\n      Lake Char­gogg­a­gogg­man­chaugg­a­gogg­chau­bun­a­gung­a­maugg\n      / Webster Lake, Webster, Massachusetts, United States of America\n    " into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.<br>Combine "\n      Lake Char­gogg­a­gogg­man­chaugg­a­gogg­chau­bun­a­gung­a­maugg\n      / Webster Lake, Webster, Massachusetts, United States of America\n    " and "" into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.